docs/do-not-merge.md

```diff
@@ -0,0 +1,466 @@
+<!--Intentionally, there's no Apache license so that the GHA fails it. This file is not meant to be merged.
+
+- https://github.com/apache/druid/pull/14266 - we removed input source security from 26 (https://github.com/apache/druid/pull/14003). Should we not include this in 27 release notes?
+
+-->
+
+# Still need
+
+search for TBD 
+
+* Caveats for array column types
+* Caching cost for upgrade notes - is it this? [#14484](https://github.com/apache/druid/pull/14484)
+* User impact of https://github.com/apache/druid/pull/14048 for highlights
+* https://github.com/apache/druid/pull/14476 seems to only be documented here in release notes and the PR?
+* Vad's input for web console section
```

Comment:
How about:

## Web console

### Console now uses the new statements API for all MSQ interaction

The console uses the new async statements API for all `sql-msq-task` engine queries.
While this has relatively few impacts on the UX of the query view you are invited to peek under the hood and check out the new network requests being sent as a working example of the new API.

There are some fun UX benefits: you can now specify `durableStorage` as the result destination for `SELECT` queries (when durable storage is configured).

<img width="655" alt="image" src="https://github.com/apache/druid/assets/177816/9399dcbd-454a-451f-bb5a-6a655025fffe">

After running a `SELECT` query that wrote its resuls to `durableStorage` you can download the full, unlimited, result set directly from the broker.

![image](https://github.com/apache/druid/assets/177816/5ca4f6f9-0728-4def-96f4-3f2b052310cf)

https://github.com/apache/druid/pull/14540
https://github.com/apache/druid/pull/14669
https://github.com/apache/druid/pull/14712

### Added UI around data source with zero replica segments

This release of Druid supports having datasources with segments that are not replicated on any historicals, the console now had special handling for datasources like that.

<img width="506" alt="image" src="https://github.com/apache/druid/assets/177816/875014ab-a4f9-4672-bf3b-df2f21d8bdfa">

[https://github.com/apache/druid/pull/14540]

### Added a Explore view

The Explore view is a simple, stateless, SQL backed, data exploration view to the web console. It lets users explore data in Druid with point-and-click interaction and visualizations (instead of writing SQL and looking at a table). This can provide faster time-to-value for a user new to Druid and can allow a Druid veteran to quickly chart some data that they care about.

![image](https://github.com/apache/druid/assets/177816/5ce191fb-5d0e-4533-b366-8f8201802f21)

The Explore view is accessible from the more button in the header.

<img width="1115" alt="image" src="https://github.com/apache/druid/assets/177816/11bfd317-f2a9-4b39-b3f6-ab710c2cafe3">

https://github.com/apache/druid/pull/14602

### Added a dialog for viewing and setting the dynamic compaction config

![image](https://github.com/apache/druid/assets/177816/d7a9bee6-ffbd-4459-8336-4b42dd229158)

[https://github.com/apache/druid/pull/14540]

### Other changes

- Replaced the **Ingestion** view with two views: **Supervisors** and **Tasks** [#14395](https://github.com/apache/druid/pull/14395)
- Added 'Start time' column to the Services tab [https://github.com/apache/druid/pull/13358]
- Added 'Replication factor' column to the Segments view [#14403](https://github.com/apache/druid/pull/14403)
- Added stateful filter URLs for all views [https://github.com/apache/druid/pull/14395]